### PR TITLE
Add job for acceptance tests

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Added a prow job for the acceptance tests

**Related issue(s)**

### CODE REVIEW

File: development/tools/jobs/acceptance_tests_test.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "prow/kyma/tests/acceptance"
+	assert.Equal(t, expName, actualPresubmit.Name)
+	assert.Equal(t, []string{"master"}, actualPresubmit.Branches)
+	assert.Equal(t, 10, actualPresubmit.MaxConcurrency)
+	assert.True(t, actualPresubmit.SkipReport)
+	assert.True(t, actualPresubmit.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPresubmit.PathAlias)
```
Comment (aszecowka) [inline development/tools/jobs/acceptance_tests_test.go]: please add an assertion that checks run_if_changed field

Comment (polskikiel) [reply development/tools/jobs/acceptance_tests_test.go]: it's already there `assert.Equal(t, "^tests/acceptance/", actualPresubmit.RunIfChanged)`#L33 :)

File: development/tools/jobs/kyma/acceptance_tests_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "kyma-tests-acceptance"
+	assert.Equal(t, expName, actualPresubmit.Name)
+	assert.Equal(t, []string{"master"}, actualPresubmit.Branches)
+	assert.Equal(t, 10, actualPresubmit.MaxConcurrency)
+	assert.True(t, actualPresubmit.SkipReport)
```
Comment (aszecowka) [inline development/tools/jobs/kyma/acceptance_tests_test.go]: please use assertion AssertThatJobRunIfChanged

Comment (polskikiel) [reply development/tools/jobs/kyma/acceptance_tests_test.go]: Done

File: development/tools/jobs/kyma/acceptance_tests_test.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	assert.True(t, actualPresubmit.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPresubmit.PathAlias)
+	tester.AssertThatHasExtraRefTestInfra(t, actualPresubmit.JobBase.UtilityConfig)
+	tester.AssertThatHasPresets(t, actualPresubmit.JobBase, tester.PresetDindEnabled, tester.PresetDockerPushRepo, tester.PresetGcrPush, tester.PresetBuildPr)
+	tester.AssertThatJobRunIfChanged(t, actualPresubmit, "tests/acceptance/fix")
+	assert.Equal(t, "^tests/acceptance/", actualPresubmit.RunIfChanged)
+	assert.Equal(t, tester.ImageGolangBuildpackLatest, actualPresubmit.Spec.Containers[0].Image)
+	assert.Equal(t, "/home/prow/go/src/github.com/kyma-project/kyma/tests/acceptance", actualPresubmit.Spec.Containers[0].Args[0])
```
Comment (jasiu001) [inline development/tools/jobs/kyma/acceptance_tests_test.go]: You can also add test for commad equal:
`assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"}, actualPresubmit.Spec.Containers[0].Command)`

Comment (polskikiel) [reply development/tools/jobs/kyma/acceptance_tests_test.go]: Done

**Pull request merged**